basics/units.ipynb

```diff
@@ -0,0 +1,403 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1050.0"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from hepunits import units as u\n",
+    "\n",
+    "1*u.meter + 5*u.cm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result is in HEP units, so mm. Indeed, for example `u.meter == 1000 == 1000 * u.mm`. \n",
+    "\n",
+    "Rather obtain the result in meters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.05"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "(1*u.meter + 5*u.cm) / u.meter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Do you need to play a bit more to get a proper feeling? This next (non-academic) exercise should help you ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **Quick time-of-flight study**\n",
+    "\n",
+    "Let's try to play with units in a meaningful way, in a kind of exercise that physicists encounter. Imagine you are investigating time-of-flight (ToF) detectors for particle identification. The time it takes a particle of velocity $\\beta = v/c= pc/E$ to travel a distance $L$ is given by\n",
+    "\n",
+    "$$\\mathrm{ToF} = \\frac{L}{c \\beta}$$\n",
+    "\n",
+    "It results that the mass $m$ of the particle can be determined from\n",
+    "\n",
+    "$$m = \\frac{p}{c}\\sqrt{\\frac{c^2 \\mathrm{ToF}^2}{L^2}-1}$$\n",
+    "\n",
+    "provided the path length $L$ and the momentum $p$ can be measured, say, by a tracking system."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What are typical ToF differences say for (charged) kaons and pions?\n",
+    "It is practical to perform the calculation as\n",
+    "\n",
+    "$$\\Delta \\mathrm{ToF} = \\frac{L}{c}(\\frac{1}{\\beta_1} - \\frac{1}{\\beta_2})\\,,$$\n",
+    "\n",
+    "with $\\frac{1}{\\beta} = \\sqrt{1+m^2c^2/p^2}$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from hepunits import c_light, GeV, meter, ps, ns\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "def ToF(m, p, L):\n",
+    "    \"\"\"Time-of-Flight = particle path length L / (c * beta)\"\"\"\n",
+    "    one_over_beta = np.sqrt(1 + m*m/(p*p))  # no c factors here because m and p given without them, hence c's cancel out ;-)\n",
+    "    return (L * one_over_beta /c_light)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For convenience, get hold of data for the proton, $K^+$ and $\\pi^+$ making use of the `Particle` package:"
```

Comment:
```suggestion
    "For convenience, get hold of data for the proton, $K^+$ and $\\pi^+$ taken from the PDG using the [`Particle` package](https://github.com/scikit-hep/particle):"
```
Particle takes the data from PDG (an important point I think, not just a "copy-paste-collection-we-needed-once-for-an-analysis") and a link to it, making it easier to check it out.

Reply:
Yes, good point.